src/Git.hs

```diff
@@ -410,6 +428,20 @@ runGit userConfig repoDir operation =
 
     DeleteTag t cont -> cont <$ callGitInRepo ["tag", "-d", refSpec t]
 
+    CheckOrphanFixups sha branch cont -> do
+      result <- let branch' = refSpec branch
+                    sha' = refSpec sha 
+                in callGitInRepo ["log", Text.unpack $ format "{}..{}" [branch',sha'], "--format='%s'"]
```

Comment:
Oh, I see what happened. On the shell, I used `--format='%s'`, to make sure that the % doesn't trigger any weird shell functionality (not sure if it would, was just precaution), but the shell will remove the quotes, so what git receives is just `--format=%s`. But when calling git this way, there is no shell to interpret it, so the `'` will get literally included in the string as part of the format.